Gemfile

```diff
@@ -23,12 +23,15 @@ gem "lucide-rails", github: "maybe-finance/lucide-rails"
 gem "stimulus-rails"
 gem "turbo-rails"
 
+# Background Jobs
+gem "good_job"
+
 # Other
 gem "bcrypt", "~> 3.1.7"
 gem "inline_svg"
 gem "jbuilder"
 gem "tzinfo-data", platforms: %i[ windows jruby ]
-gem "money-rails", "~> 1.12"
```

Comment:
could you please explain the reasoning behind not using this gem?

Reply:
No major benefit to using it.

Reply:
https://github.com/maybe-finance/maybe/blob/4d5d35b277ce89002f02b3a63d3380abdcc384e7/db/migrate/20240206031739_replace_money_field.rb#L3

This migration fails without it!

Reply:
I had this issue, It should be fixed by #438